runtime/src/snapshot_utils.rs

```diff
@@ -39,15 +40,28 @@ use {
     thiserror::Error,
 };
 
+/// Information about a snapshot archive: its path, slot, hash, and archive format
+pub type SnapshotArchiveInfo = (PathBuf, (Slot, Hash, ArchiveFormat));
+
+/// Information about an incremental snapshot archive: its path, full snapshot base slot, incremental snapshot slot, hash, and archive format
+pub type IncrementalSnapshotArchiveInfo = (PathBuf, (Slot, Slot, Hash, ArchiveFormat));
```

Comment:
I think this should be a struct with named fields

Reply:
I agree. Will do!

Reply:
Done!